assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -311,6 +315,34 @@ export const selectors = {
 		return referenceSiteURL;
 	} ),
 
+	/**
+	 * Gets the current reference URL to use.
+	 *
+	 * This selector should be used to get the contextual URL for requesting
+	 * URL-specific data from Google APIs.
+	 *
+	 * If a current entity exists, this will return the same value as the
+	 * `getCurrentEntityURL` selector. Otherwise it will fall back to returning
+	 * the same value as the `getReferenceSiteURL` selector.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(string|undefined)} The current reference URL, or undefined if
+	 *                              not loaded yet.
+	 */
+	getCurrentReferenceURL: createRegistrySelector( ( select ) => () => {
+		const { currentEntityURL, referenceSiteURL } = select( STORE_NAME ).getSiteInfo() || {};
```

Comment:
I feel like maybe we should use the dedicated selectors for these, rather than reaching for them directly from the main site info selector. This is what the doc says above but that would be a bit more "guaranteed" if we used their respective selectors. It would still only invoke the underlying resolver once so I don't think there's a performance penalty to do so. What do you think?

Reply:
Good point, that suggestion is also cleaner in not replicating the internal details of where to access the respective value from.